pkg/sql/sem/builtins/builtins.go

```diff
@@ -3566,7 +3571,9 @@ func hashBuiltin(newHash func() hash.Hash, info string) builtinDefinition {
 			ReturnType: tree.FixedReturnType(types.String),
 			Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
 				h := newHash()
-				feedHash(h, args)
+				if feedHash(h, args) == 0 {
```

Comment:
is `md5('')`? tested anywhere? gotta make sure that still works.

Reply:
Oh yeah, hmm, I broke that. Let me rework. Thanks.